docs/README_sp.md

```diff
@@ -0,0 +1,195 @@
+
+Ejecutar `make cluster`
+
+### 4. Agregar un propietario verificado para el dominio
+
+Agregue la siguiente cuenta de servicio como [additional verified owner][]:
+
+`cnrm-system@${PROJECT_ID}.iam.gserviceaccount.com`
+
+donde `${PROJECT_ID}` se reemplaza por su ID de proyecto de Google Cloud.
+
+### 5. Build y desplegar
+
+Ejecute `make build-all`.
+
+## Probar la Aplicación
+
+Una vez que se despliega su aplicación, puede probarla navegando a `https://$DOMAIN`,
+donde `$DOMAIN` es el dominio personalizado que configuró `env.mk`.
+
+## Actualizar la Aplicación
+
+Ejecutar `make build-all` hará el build y desplegará la aplicación, incluidos los cambios realizados en la infraestructura. Tenga en cuenta que eliminar recursos de `infrastructure-tpl.yaml`
+no hará que se eliminen. Debe ejecutar `make delete` antes de eliminar el recurso (luego volver a implementar con `make build-all` después de eliminarlo), o eliminar manualmente el recurso con `kubectl delete`.
+
+```shell
+# construye e implementa infraestructura de back-end, frontend e KCC
+make build-all
+
+# construye e implementa solo el servicio de back-end Go
+make build-backend
+
+# construye y despliega solo la aplicación web angular frontend
+make build-webui
+```
+
+## Limpieza
+
+La ejecución de `make delete` eliminará los recursos del conector de configuración de su clúster, lo que hará que el conector de configuración elimine los recursos de GCP asociados. Sin embargo, debe eliminar manualmente su servicio Cloud Run y ​​GKE Cluster.
+
+[APIs & Services -> Credentials]: https://console.cloud.google.com/apis/credentials
+[Cloud Build]: https://cloud.google.com/cloud-build/docs
+[Config Connector]: https://cloud.google.com/config-connector/docs
+[Cloud DNS Managed Zone]: https://cloud.google.com/dns/zones
+[update name server records]: https://cloud.google.com/dns/docs/migrating#update_your_registrars_name_server_records
+[domain ownership verification]: https://cloud.google.com/storage/docs/domain-name-verification#verification
+[additional verified owner]: https://cloud.google.com/storage/docs/domain-name-verification?_ga=2.256052552.-234301672.1582050261#additional_verified_owners
+[Enable Identity Platform]: https://console.cloud.google.com/marketplace/details/google-cloud-platform/customer-identity
+[Identity Platform -> Providers]: https://console.cloud.google.com/customer-identity/providers
+[Identity Platform quickstart guide]: https://cloud.google.com/identity-platform/docs/quickstart-email-password#sign_the_user_in
+[Identity Platform page in the GCP console]: https://console.cloud.google.com/marketplace/details/google-cloud-platform/customer-identity
+[OAuth consent screen]: https://console.cloud.google.com/apis/credentials/consent
+[Identity Platform -> Settings]: https://console.cloud.google.com/customer-identity/settings
+[Setting up OAuth 2.0 guide]: https://support.google.com/cloud/answer/6158849?hl=en
+[configurar gcloud]: https://cloud.google.com/sdk/docs
```

Comment:
I take back my suggestion from the last review:
```suggestion
[set up gcloud]: https://cloud.google.com/sdk/docs
```

I think it's better to use the same link reference names and just translate where they appear.

Reply:
I'm actually not sure I agree with this -- per my suggestion above, it would be good to localize the links themselves to present the Spanish-language versions where possible. In that case, the links are different than the original README and therefore it may not make as much sense to maintain the original English reference name.

Reply:
If the links are localized I agree. However, it looks like, based on me playing around with my Google Cloud console and GCP documentation settings, the default language is whatever you're using, which for Spanish users reading the README it's likely to be Spanish.

Reply:
I've decided to rely on GCP's implicit user/cookie based localization. It should make it easier to update links as necessary